src/main/java/Activities.java

```diff
@@ -0,0 +1,34 @@
+public class Activities {
+    private String Activity;
+    private int status = 0; // 0 represent undone: 1 represents done
+
+    public Activities(String activity) {
+        Activity = activity;
+    }
+    public static Activities newActivity(String activity){
+        return new Activities(activity);
+    }
+
+    public void doIt() {
+        status = 1;
+    }
```

Comment:
Could the method name be changed to doActivity() to reduce the ambiguity from using 'it' instead?